python/GafferUI/EditScopeUI.py

```diff
@@ -144,6 +144,7 @@ def __init__( self, plug, **kw ) :
 		# run the default dropSignal handler from PlugValueWidget.
 		self.dropSignal().connectFront( Gaffer.WeakMethod( self.__drop ) )
 
+		self.__updateLabelVisibility()
```

Comment:
For completeness, we should probably override `_updateFromMetadata()` to update label visibility too.

Reply:
Good idea! e0717480564c1630ae0b0c90b5b532b503f1b4f2